README.md

```diff
@@ -162,14 +162,16 @@ Device information
 | u8 | device_revision | device-specific hardware revision |  |
 | u8 | firmware_version_major | Firmware version major number. |  |
 | u8 | firmware_version_minor | Firmware version minor number. |  |
-| u8 | firmware_version_patch | Firmware version patch number. |  |
+| u16 | firmware_version_patch | Firmware version patch number. |  |
 
 #### 5 protocol_version
 The protocol version
 
 | Type | Name             | Description                                                      | Units |
 |------|------------------|------------------------------------------------------------------|-------|
-| u32 | protocol_version | The protocol version |  |
+| u8 | version_major | Protocol version major number. |  |
+| u8 | version_minor | Protocol version minor number. |  |
+| u16 | version_patch | Protocol version patch number. |  |
```

Comment:
Maybe we should use the reserved idea, is there a motive to use 16bits for `version_path` ?